api/node-echoprint-server/controllers/api.js

```diff
@@ -1,42 +1,52 @@
-var urlParser = require('url');
-var log = require('winston');
-var fingerprinter = require('./fingerprinter');
-var server = require('../server');
-var config = require('../config');
+const urlParser = require('url');
+const log = require('winston');
+const fingerprinter = require('./fingerprinter');
+const server = require('../server');
+const config = require('../config');
 
 /**
  * Querying for the closest matching track.
  */
 exports.query = function(req, res) {
-  var url = urlParser.parse(req.url, true);
-  var code = url.query.code;
-  if (!code)
-    return server.respond(req, res, 500, { error: 'Missing code' });
+  const url = urlParser.parse(req.url, true);
+  const code = url.query.code;
+  if (!code) {
+    res.status(400)
+       .send({ error: 'Missing code' });
+  }
 
-  var codeVer = url.query.version;
-  if (codeVer != config.codever)
-    return server.respond(req, res, 500, { error: 'Missing or invalid version' });
+  const codeVer = url.query.version;
```

Comment:
Can we not use abbreviated variable names, we could have something like `codeVersion`.